GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -716,6 +723,32 @@ private synchronized int writeInstanceData(Sheet sheet, final int startRow,
 
         return maxRow + 1;
     }
+    
+    private String constructDisplayName(
```

Comment:
But then we need a lot of conversion as it uses Question and QAS object lists, and here we have QuestionDto and String lists.